examples/access-graph/modules/config/teleport-config/main.tf

```diff
@@ -0,0 +1,84 @@
+# Generate the Teleport YAML configuration
+locals {
+  https_keypairs_section = <<EOT
+  https_keypairs:
+    - key_file: certs/teleport-key.pem
+      cert_file: certs/teleport.pem
+EOT
+
+  # SSH service enabled value
+  ssh_enabled = var.enable_ssh_service ? "yes" : "no"
+
+  # Access graph enabled
+  access_graph_enabled = var.enable_access_graph ? "true" : "false"
+
+  # Audit log enabled
+  audit_log_enabled = var.enable_audit_log ? "true" : "false"
```

Comment:
nit: i believe you can just use the bool directly in the `teleport_yaml` without manual conversion.